spec/furniture/marketplace/carts_controller_request_spec.rb

```diff
@@ -7,7 +7,7 @@
 
   describe "#update" do
     subject(:perform_request) do
-      put polymorphic_path(cart.location), params: {cart: {delivery_address: "123 N West St"}}
+      put polymorphic_path(cart.location), params: {cart: {delivery_address: "123 N West St", contact_phone_number: "(415)-123-4567"}}
```

Comment:
It would be nice if there was a `specify` block that asserted this actually changed.

Reply:
So good!